docs/src/main/asciidoc/sqs.adoc

```diff
@@ -806,6 +806,94 @@ NOTE: The same factory can be used to create both `single message` and `batch` c
 
 IMPORTANT: In case the same factory is shared by both delivery methods, any supplied `ErrorHandler`, `MessageInterceptor` or `MessageListener` should implement the proper methods.
 
+==== Automatic Batching with AWS SDK
+
+Spring Cloud AWS supports automatic message batching using AWS SDK's `SqsAsyncBatchManager`. This feature optimizes SQS operations by automatically batching requests under the hood to improve performance and reduce AWS API calls.
```

Comment:
How about we call out earlier that this feature is mostly intended to be used with the `SqsTemplate` rather than a general client to be used as a default bean, including in `@SqsListener`?